JamesRexMiller4/refactortractor
# Fetch1 gi

## What Does this PR do?
 - This PR will implement the 'GET' portion of our fetch for all our data. It is bundled into a promise all in order to prevent one dataset populating before any of the other datasets and breaking everything. Essentially the promise all just says 'wait till we have all the data before firing.' I have also added global variables for each class so we can assign  a new instantiation in the promise all. The generate randomUser function has also been included in the promise all.

## Where should you start?
 -  Lines 31-64 in scripts should be all of the fetch. 

## How should this be manually tested?
 - For now this should not break anything. You can add a console.log for any of the datasets to make sure they are properly being called

## Any background context you want to provide?\
 - I am very open to any suggestions or changes for the catch and error message it includes.

## This should resolve #2 

@JamesRexMiller4 @RayRedGoose 

Thanks!
